contrib/Shopify/sarama/headers.go

```diff
@@ -2,7 +2,8 @@ package sarama
 
 import (
 	"github.com/signalfx/signalfx-go-tracing/ddtrace/tracer"
-	sarama "gopkg.in/Shopify/sarama.v1"
+
+	"github.com/Shopify/sarama"
```

Comment:
essentially this change means we don't have an instrumentation for `"gopkg.in/Shopify/sarama.v1"` anymore? Or can someone importing from gopkg.in still use this instrumentation? Are they the same package with different import paths or are they different versions?

Reply:
>essentially this change means we don't have an instrumentation for "gopkg.in/Shopify/sarama.v1" anymore?

true. it is deprecated

> Or can someone importing from gopkg.in still use this instrumentation?

yes. e.g. using `v1.6.2` version

>Are they the same package with different import paths or are they different versions?

As far as I understand Shopify/sarama.v1 stopped to use gopkg.in after they moved to Go Modules